src/app/pages/FrontPage/index.jsx

```diff
@@ -33,34 +19,7 @@ import CanonicalAdBootstrapJs from '#containers/Ad/Canonical/CanonicalAdBootstra
 import LinkedData from '#containers/LinkedData';
 import ATIAnalytics from '#containers/ATIAnalytics';
 import ChartbeatAnalytics from '#containers/ChartbeatAnalytics';
-
-export const StyledFrontPageDiv = styled.div`
-  /* To add GEL Margins */
-  margin: 0 ${GEL_MARGIN_BELOW_400PX};
-  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    margin: 0 ${GEL_MARGIN_ABOVE_400PX};
-  }
-
-  /* To add extra spacing */
-  padding-top: ${GEL_SPACING};
-  padding-bottom: ${GEL_SPACING_QUAD};
-
-  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    padding-top: ${GEL_SPACING_DBL};
-  }
-
-  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-    padding-bottom: ${GEL_SPACING_TRPL};
-  }
-
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    padding-top: 0;
-  }
-
-  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
-    padding-bottom: ${GEL_SPACING_QUIN};
-  }
-`;
+import PageContainer from '#lib/pageStyles/PageContainer';
```

Comment:
Hey - so the file is still called `pageContainer`, but the imports are referencing `PageContainer` (if you've been using a mac, this will still work as the filesystem is case insensitive, but this import will fail on a case sensitive system like Linux, I haven't checked but the build should have failed because of this).

Reply:
Updated file name of PageContainer to match component case